draco/analysis/transform.py

```diff
@@ -453,8 +459,13 @@ def process(self, sstream):
         # variance for the m-modes
         weight_sum = sstream.weight[:].sum(axis=-1)
 
+        if self.telescope is not None:
+            setmmax = self.telescope.mmax
```

Comment:
This variable name doesn't make sense. `setxyz` sounds more like a binary parameter that sets some behaviour, whereas it actually represents the `mmax` itself.